hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -89,3 +95,20 @@ See the [bootmodes documentation](README.md) for the boot sequence and alternati
 
 ## Special bootcode.bin-only boot mode (not Pi 4B, CM4 and Pi 400)
 If you are unable to use a particular USB device to boot your Raspberry Pi, an alternative for the Pi 2B v1.2, 3A+, 3B and 3B+ is to use the special bootcode.bin-only boot mode as described [here](README.md). The Pi will still boot from the SD card, but `bootcode.bin` is the only file read from it.
+
+## Hardware compatibility for USB mass storage devices
+Before attempting to boot from a USB mass storage device it is advisable to verify that the device works correctly under Linux. Boot using an SD card and plug in the USB mass storage device. This should appears as a removable drive.
+
+*Spinning hard-disk drives nearly always require a powered USB hub. Even if it appears to work you are likely to encounter intermittent failures without a powered USB HUB*
+
+This is especially important with USB SATA adapters which may be supported by the bootloader in mass storage mode but fail if Linux selects [USB Attached SCSI - UAS](https://en.wikipedia.org/wiki/USB_Attached_SCSI) mode.
+
+See this [forum thread](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) about UAS and how to add [usb-storage.quirks](https://www.kernel.org/doc/html/v5.0/admin-guide/kernel-parameters.html) to workaround this issue.
+
+## Multiple bootable drives
+When searching for a bootable partition the bootloader scans all USB mass storage devices in parallel and will select the first to respond. If the boot partition does not contain a suitable start.elf file the next available device is selected.
+
+As with earlier Raspberry Pi models there is no method for specifying the boot device according to the USB topology because this would slow down boot and adds unnecessary and hard to support configuration complexity.
```

Comment:
I guess this "As with earlier Raspberry Pi models" needs rewording, now that this is on the "generic" msd.md page and not on a Pi4-specific page?

Reply:
Removed "As with earlier Pi models"